frontend/lib/norent/homepage.tsx

```diff
@@ -1,19 +1,364 @@
+              We make it easy to weigh your options and notify your landlord by
+              email or by certified mail for free.
+            </p>
+            <br />
+          </div>
+          <div className="container jf-wide-container jf-how-it-works-container">
+            <div className="columns is-variable is-8-desktop">
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("chatbox")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    8 Steps
+                  </p>
+                  <p>
+                    Answer a few questions about yourself and your landlord or
+                    management company.
+                  </p>
+                </div>{" "}
+              </div>
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("connection")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    Legal Protections
+                  </p>
+                  <p>
+                    Our letter cites the most up-to-date legal ordinances that
+                    protect tenant rights in your state.
+                  </p>
+                </div>
+              </div>
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("letters")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    Free Certified Mail
+                  </p>
+                  <p>
+                    After you’ve reviewed your letter, we send it to your
+                    landlord on your behalf by email and by certified mail.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <br />
+            <span className="is-hidden-mobile">
+              <BuildMyLetterButton />
+            </span>
+          </div>
+        </div>
+      </section>
+
+      <section className="hero">
+        <div className="hero-body">
+          <div className="container jf-tight-container has-text-centered jf-space-below">
+            <h2 className="title is-spaced">Legally vetted</h2>
+            <p className="subtitle is-size-4">
+              Our free letter builder was built with{" "}
+              <OutboundLink
```

Comment:
Thanks for remembering to make this an `<OutboundLink>` and not an `<a>`, I often forget to do that myself!